src/modules/tracker.js

```diff
@@ -516,7 +516,7 @@ class Tracker {
     if (term && typeof term === 'string') {
       // Ensure parameters are provided (required)
       if (parameters && typeof parameters === 'object' && !Array.isArray(parameters)) {
-        const url = `${this.options.serviceUrl}/autocomplete/${helpers.encodeURIComponentRFC3986(helpers.trimNonBreakingSpaces(term))}/search?`;
```

Comment:
Do we want to call `normalizeSpaces` here? (Without the trim)

Reply:
excellent catch once again